bin/crab.py

```diff
@@ -149,11 +149,11 @@ def log_exception(exc_type, exc_value, tback):
         client()
         exitcode = 0 #no exceptions no errors
     except RESTInterfaceException as err:
+        exitcode=err.exitcode
```

Comment:
I though that this syntad is not python3 compatible, i.e. exceptions do not have members anymore. But maybe I am confused, since I guess you tested. But for my educatin, @mapellidario can you chime in ?

Reply:
Doesn't this `exitcode` come from

https://github.com/dmwm/CRABClient/blob/e3741f8dfe02a39e972013f85d54866f6e34732f/src/python/CRABClient/ClientExceptions.py#L98-L102

?

If this is the case, then there is nothing to worry about, this is ok in both py2 and py3.

Reply:
yes, you are right

Reply:
thanks Dario and Daina